### ISSUE TITLE
[Bug]: plugin not working on ipados 18.0.1 and ios 18.0.1

### ISSUE THREAD

**Issue by laurentbonet** [st3v3nmw/obsidian-spaced-repetition#1149]
### Describe the bug

Strange display for card review see screen dump
UI elements seems to be in column rather than line

### To Reproduce

Activate review flascard command

### Expected behavior

Normal display of the review card box

### Screenshots

<img width="880" alt="Capture d’écran   2024-10-21 à 11 34 34 jpeg" src="https://github.com/user-attachments/assets/90f564eb-c3dc-4cef-9007-874995a473aa">

### OS of your device

Ipad os18.0.1 ios 18.0.1

### Obsidian version

1.7.4

### Plugin version

1.13.2

**Comment by Takaishisama** [comment]
Yes, it also happens with the android version in both mobile and tablets even if the height and width are set to 100%. Only workaround I've found is to either use the device vertically (quite awkward for a tablet) or disable the "Show context in cards" option under User interface > Flashcards. I find this option quite handy, so it's a pity to have to disable it until it's fixed.

**Comment by jackchealdang** [comment]
same issue here iPadOS 18.1

**Comment by KyleKlus** [comment]
Hi, this one should be fixed now 😊
